apps/molgenis-components/README.md

```diff
@@ -36,3 +36,8 @@ This module helps developers create and share vue components for use with the mo
 - build and test your component
 - add a sample to your sfc by using the ```<docs></docs>``` tag
 - share component by adding it the lib/main.js export, and running the build
+
+
+# FAQ
+
+If there are no components showing on first install, remove the ```gen-docs``` folder
```

Comment:
should not be need , how to reproduce ?

Reply:
clean install of emx2, run yarn and then yarn dev in molgenis components

Reply:
can not reproduce , its a local dev issue , suggest you create an issue with steps to reproduce , strange thing is the first thing yarn dev does is remove the gen-docs folder , maybe this does not happen on some systems but i cant reproduce this on my system, so suggest to leave the comment ( maybe as a note instead of FAQ :) ) and submit an issue